code/command_block/command_block.py

```diff
@@ -0,0 +1,70 @@
+import os
+from time import sleep
+import subprocess
+import configparser
+
+lastLine = None
+sensor_logs = "foo"
+text_buffer = "buffer.txt"
```

Comment:
these should be configurable from [argparse module](https://towardsdatascience.com/learn-enough-python-to-be-useful-argparse-e482e1764e05)

Reply:
what about configparser?

Reply:
Solved with conffig file